packages/contracts-bedrock/test/vendor/Initializable.t.sol

```diff
@@ -241,10 +241,18 @@ contract Initializer_Test is CommonTest {
                 )
             })
         );
-        // SoulGasToken
+        // SoulGasTokenImpl
         contracts.push(
             InitializeableContract({
-                name: "SoulGasToken",
+                name: "SoulGasTokenImpl",
+                target: address(soulGasToken),
```

Comment:
Should be `EIP1967Helper.getImplementation(address(soulGasToken))`?

Reply:
fixed